docs/src/components/ColorSwatchFamily/ColorSwatchFamily.component.tsx

```diff
@@ -0,0 +1,57 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import styled from 'styled-components';
+import { ColorSwatch } from '../ColorSwatch';
+import { Typography } from '../../../../src/Typography';
+// THEME
+import { colors } from '../../../../src/theme';
+
+
+// INTERFACES
+interface ColorSwatchFamilyProps {
+  baseColorHexCode: string;
+  baseColorName: string;
+  darkColorHexCode?: string;
+  lightColorHexCode?: string;
+}
+
+
+// STYLES
+const StyledColorSwatchFamily = styled.div`
+  margin: 1rem 0;
```

Comment:
`box-sizing: border-box;`

Reply:
How are you determining which components need this?

Reply:
They all do, but containing components or anything that has padding/margins added usually catch my eye.

Reply:
@sharkparty sounds good. Where's the sauce?